daemon/main.go

```diff
@@ -110,6 +111,12 @@ var RootCmd = &cobra.Command{
 }
 
 func main() {
+
+	// Open socket for using gops to get stacktraces of the agent.
+	if err := gops.Listen(gops.Options{}); err != nil {
+		log.Fatal(err)
```

Comment:
Can you change this to `log.WithError(err).Fatal("Unable to initialize gops")`? I assume the error returned by  `Listen()` does not include that this is about gops?

Reply:
Did you test that logging is set up properly at this point?

Reply:
It logs to `stderr`, but I will have the error case mimic the behavior of if the execution of the following command fails:
```
... err != nil {
  fmt.Println(err)
  os.Exit(-1)
}
```